test/Fixtures/Fixtures.yaml

```diff
@@ -2458,3 +2458,170 @@
       StartDate: ~
       LocalStartDate: 2022-10-02T08:00:00
     FrontierLabsLogFileExtractor:
+
+- Name: "Partial file 3.14 - empty"
+  Record:
+    Extension:
+    Stem: data
+    StartDate:
+    LocalStartDate:
+    TrueStartDate:
+    TrueEndDate:
+    # this is a faulty file FL011 wrapping FL003
```

Comment:
The major reason I use YAML is because it allows comments (and because it is way less verbose than JSON)

Comments are allowed pretty much anywhere they would be in other languages.

I commented this case because these are seriously whacked numbers. 22 samples divided by the ≈158,646,272 actual samples in the data block of the wave file results in a reported sample rate of 7,211,194 Hz (7.2 megahertz). This is what happens when https://github.com/ecoacoustics/known-problems/blob/main/frontier_labs/FL003.md occurs. 

Anyway, just trying to denote that this is not a normal case.